### ISSUE TITLE
Cursor with array

### ISSUE THREAD

**Issue by khankuan** [Yomguithereal/baobab-react#88]
Hi, I noticed that the cursors in the examples are wrapped in an array. Is there a reason for that? Thanks!

**Comment by Yomguithereal** [comment]
Hello @khankuan. Can you precise which examples you are talking about please? I am not sure to know which ones you are mentionning.

**Comment by khankuan** [comment]
Something like this:

```
export default branch(List, {
  cursors: {
    colors: ['colors']
  }
});
```

**Comment by Yomguithereal** [comment]
You'd like to do this?

```jsx
export default branch(List, {
  cursors: {
    colors: 'colors'
  }
});
```

If so, this is linked to #86. I need to make this happen.

**Comment by khankuan** [comment]
Ah yes :+1:

**Comment by Yomguithereal** [comment]
Well I just tested that and it seems you can already do it. I usually stick to the arrays because it's easier to change in the future if needed.

**Comment by khankuan** [comment]
I tried that and got .some is undefined :/

**Comment by Yomguithereal** [comment]
Can you give me the full error message please? Plus your version of Baobab/baobab-react.

**Comment by khankuan** [comment]
```
Uncaught TypeError: path.some is not a function
```

which points to
```
type.dynamicPath = function (path) {
  return path.some(function (step) {
    return type['function'](step) || type.object(step);
  });
};
```

On baobab 2.2.2 and baobab-react 1.0.0

**Comment by Yomguithereal** [comment]
Ok, found the bug. It's in Baobab. Fixing it.

**Comment by Yomguithereal** [comment]
Just published baobab v2.2.1. This should be good with this version.

**Comment by khankuan** [comment]
Nice!
